frappe/public/js/frappe/misc/tools.js

```diff
@@ -77,6 +77,11 @@ frappe.slickgrid_tools = {
 	get_filtered_items: function(dataView) {
 		var data = [];
 		for (var i=0, len=dataView.getLength(); i<len; i++) {
+		    // not_quoted to remove single quotes at start and end of total labels
+		    var not_quoted = dataView.getItem(i).account_name;
```

Comment:
cannot be hard coded for `account_name`.

Also please use tabs, the project uses tabs unfortunately, so we have to follow!